DirectEmployers/MyJobs
# Fix sitemap sharding.

## What's this PR do?
Slightly alters sitemap sharding. See ticket PD-3426 for why this is being done.
Sends an email if the sharding is getting close to being wrong again, that way we're not relying on third parties to tell us about this.
Fixes settings because somehow my draft of the environment switching got committed, rather than the final change.

## Where should the reviewer start?
Double-check sitemap_views.py

## How should this be manually tested?
Go to any sitemap. Confirm it doesn't have exactly 12500 jobs but does have jobs that belong on that sitemap.

## Any background context you want to provide?
Right now we're bumping up against a limit on some pages.
